src/tigerbeetle/cli.zig

```diff
@@ -107,6 +110,12 @@ const CliArgs = union(enum) {
         \\        (Total RAM) - 3GB (TigerBeetle) - 1GB (System), eg 12GB for a 16GB machine.
         \\        Defaults to 1GB.
         \\
+        \\  --lsm-manifest-memory=<size><KB|MB|GB>
+        \\        Sets the amount of memory allocated for LSM-tree manifests. When the
+        \\        number or size of LSM-trees would become too large for their manifests to fit
+        \\        into memory, such requests are rejected.
+        \\        Defaults to 64MB.
```

Comment:
Since this is a preexisting issue I'd prefer to fix them both in a separate PR and have filed this issue about it https://github.com/tigerbeetle/tigerbeetle/issues/1466